subjects/git/audit/README.md

```diff
@@ -0,0 +1,62 @@
+
+##### Setup and Installation
+
+###### Has the student installed Git on their local machine?
+
+###### Has the student properly configured Git with a usename and email address?
+
+##### Git commits to commit
+
+###### Has the file `hello.rb` been created in the proper workspace?
+
+###### Is Git initialized in the workspace with the `git init` command?
+
+###### Did the student use `git status` to check the status of the working directory?
+
+###### Has `hello.rb` file been staged and committed with the appropriate commands `git add` and `git commit`?
+
+##### Git History & Logging
+
+###### Did the student display the Git history of the working directory with `git status` command?
+
+###### Was the student able to display the history in various ways, such as showing only the hashes and commit messages, or controlling the number of entries displayed?
+
+###### Did the student customize the format of Git logs and display them accordingly with `git log --pretty=format:'%h %ad | %s%d [%an]' --graph --date=short`?
+
+##### Checking Out & Tagging
+
+###### Did the student make, revert, and stage changes efficiently in response to the provided instructions?
+
+###### Did the student add the 'oops' tag and remove commits following 'v1' as specified?
+
+###### Did the student display logs with the deleted commits or without them as instructed?
+
+##### Branching, Merging & Rebasing
+
+###### Did the student create and switch to a new branch named 'greet'?
+
+###### Did the student create a new file, stage it, and make a commit in the 'greet' branch?
+
+###### Did the student successfully merge changes from 'main' into 'greet' and resolve any conflicts that arose?
+
+###### Did the student successfully rebased 'greet' branch on top of 'main' branch?
+
+###### Did the student switch branches, update 'hello.rb' in the 'main' branch, and understand the differences between versions?
+
+###### Did the student understand clearly the differences between rebasing and merging?
```

Comment:
in questions like this you need to specify that the auditer needs to ask the student that question... like:
```
##### Ask the student about the differences between rebasing and merging
###### Did the student understand clearly the differences between rebasing and merging?

```